collector/src/descriptor/kind/bridge_server_descriptor.rs

```diff
@@ -111,6 +250,28 @@ fn take_uniq<'a>(
     }
 }
 
+fn take_multi_descriptor_lines<'a>(
+    map: &'a mut HashMap<&str, Vec<DescriptorLine>>,
+    key: &str,
+    len: usize,
+) -> Result<Vec<DescriptorLine<'a>>, Error> {
+    if let Some(v) = map.remove(key) {
+        let format_ok = v.iter().fold(true, |acc, e| {
```

Comment:
this is hard to read, use [`all`](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.all) or [`any`](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.any) instead please